project-react/src/components/Sidebar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import {SidebarData} from '../data/SidebarData'
+import { NavLink } from "react-router-dom";
+
+const Sidebar = () => {
+    const activeLink = 'hover:bg-red-500 mt-7 pl-7 w-full h-14 flex justify-start items-center text-white text-2xl space-x-1 font-bold bg-red-500'
+    const normalLink = 'hover:bg-red-500 pl-7 mt-7 w-full h-14 flex justify-start items-center text-white text-2xl space-x-1 font-bold'

```

Comment:
could u make constants file and add those constants to it , and then import those values here ?

Reply:
Yes I can and i did it.